### PR TITLE
Fix datacenter documentation

### PR DESCRIPTION
**Pull request by schmichael**
Thanks to @magiconair on #1641

### CODE REVIEW

File: website/source/docs/cluster/bootstrapping.html.md [hunk 1]
```diff
@@ -54,6 +54,11 @@ servers and clients. A suggested base configuration is:
 # Name the region, if omitted, the default "global" region will be used.
 region = "europe"
 
+# Clients will connect to servers in other datacenters as a backup if no
+# local servers can be found. It's highly recommended to run servers in each
+# datacenter.
+datacenter = "dc1"
+
```
Comment (sean-) [inline website/source/docs/cluster/bootstrapping.html.md]: This shouldn't be necessary when using Consul.

Comment (dadgar) [inline website/source/docs/cluster/bootstrapping.html.md]: Yeah and not generally applicable either actually

Comment (schmichael) [inline website/source/docs/cluster/bootstrapping.html.md]: Sounds good, removed.

**Comment by dadgar** [comment]
@schmichael I think you can just close this PR. Clients should set their DC

**Pull request closed without merging**